io.github.nostar.DroidStar.yml

```diff
@@ -0,0 +1,40 @@
+app-id: io.github.nostar.DroidStar
+runtime: org.kde.Platform
+runtime-version: 5.15-22.08
+sdk: org.kde.Sdk
+command: droidstar
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --socket=pulseaudio
+  - --device=dri
+  - --filesystem=xdg-download
+cleanup:
+  - /man
+  - /include
+  - /share/man
+  - /share/doc
+  - /doc
+  - "*.a"
+  - "*.la"
+modules:
+  - name: imbe_vocoder
+    buildsystem: simple
+    build-commands:
+      - make
+      - install -Dm644 libimbe_vocoder.a /app/lib/libimbe_vocoder.a
+    sources:
+      - type: archive
+        url: https://github.com/nostar/imbe_vocoder/archive/refs/heads/master.zip
```

Comment:
I will repeat: no branch. Tricking it by downloading the `master` archive doesn't cut it. You need to pin a specific commit or tag.

Reply:
I don't know.. A few years ago when I made manifestos for flatpak we didn't use gitbub ... Between some new changes and using GitHub for everything costs me twice as much.. I don't just have to see the changes but try to use GitHub which is complicated

Reply:
Here on Flathub we have a bot that can automatically create new Pull Requests with updates when a new version of any component of the Flatpak gets released so you don't have to update it manually, just test and merge the PR. :-) See [this](https://github.com/flathub/flatpak-external-data-checker#on-flathub) for more info.